[android] Need Fix the readme

My environment is gradle 2.4 on Android.   The `ReactMaterialKitPackage` should be added in the `MainApplication.java` file. 

Moreover, I encountered errors like

 `:react-native-material-kit:prepareReleaseDependencies FAILED FAILURE: Build failed with an exception. * What went wrong:...`.  

I fix this problem by editing some lines of  `node_modules/react-native-material-kit/android/build.gradle` from `dependencies {
    provided 'com.facebook.react:react-native:0.+'
}` to `dependencies {
    compile 'com.facebook.react:react-native:0.+'
}`

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -30,5 +30,5 @@ repositories {
 }
 
 dependencies {
-    provided 'com.facebook.react:react-native:0.+'
+    compile 'com.facebook.react:react-native:0.+'
 }

